sdks/java/io/bigquery-io-perf-tests/src/test/java/org/apache/beam/sdk/bigqueryioperftests/BigQueryIOIT.java

```diff
@@ -96,7 +96,7 @@ public static void setup() throws IOException {
     metricsBigQueryDataset = options.getMetricsBigQueryDataset();
     metricsBigQueryTable = options.getMetricsBigQueryTable();
     testBigQueryDataset = options.getTestBigQueryDataset();
-    testBigQueryTable = options.getTestBigQueryTable();
+    testBigQueryTable = options.getTestBigQueryTable() + TEST_ID;
```

Comment:
Nit: Maybe consider adding a "_" before TEST_ID (for readability purposes in BigQuery). I mean something like this: 

`String.format("%s_%s", options.getTestBigQueryTable(), TEST_ID); `